galasa-extensions-parent/dev.galasa.extensions.mocks/src/main/java/dev/galasa/extensions/mocks/MockFrameworkInitialisation.java

```diff
@@ -33,6 +33,7 @@ public class MockFrameworkInitialisation implements IApiServerInitialisation {
     private MockFramework framework;
     
     protected URI authStoreUri;
+    protected URI usersStoreUri;
```

Comment:
The users database is in the auth store, so I don't see this usersStoreUri being used unless we create a separate store